src/caret_analyze/plot/bokeh/plot_factory.py

```diff
@@ -242,3 +243,39 @@ def _create_communication_period_plot_tuple(
         *communications: Communication
     ) -> CommunicationTimeSeriesPlot:
         return CommunicationPeriodPlot(communications)
+
+    @singledispatchmethod
+    def create_response_time_histogram_plot(arg) -> TimeSeriesPlot:
+        """
+        Get ResponseTimePlot instance.
+
+        Parameters
+        ----------
+        path : Collection[Path]
+            Target path.
+            This also accepts multiple path inputs by unpacking.
+
+        Returns
+        -------
+        ResponseTimePlot
+
+        """
+        raise InvalidArgumentError(f'Unknown argument type: {arg}')
+
+    @staticmethod
+    @create_response_time_histogram_plot.register
+    def _create_response_time_histogram_plot(
+        paths: Collection[Path],
+        case: str,
+        binsize_ns: int
```

Comment:
Error occurs.

![image](https://user-images.githubusercontent.com/19860128/198505940-37fab160-7f59-4684-906a-89fb90aacc5a.png)

```suggestion
        paths: Collection[Path],
        case: str = 'best-to-worst',
        binsize_ns: int = 10000000
```
It seems default values are needed.